### PR TITLE
Show toolbar on meta key press

### PR DESCRIPTION
**Pull request by ellatrix**
Start for #552. Ideally the focus would also move to the first button of the toolbar.

### CODE REVIEW

**Comment by iseulde** [comment]
@youknowriad How do we best move focus to the toolbar in React?

**Comment by youknowriad** [comment]
@iseulde There's no special React feature for this, you'll have to store the ref and call `.focus`.

**Comment by iseulde** [comment]
So, to clarify, the goal of this PR is to ignore the toolbars entirely in the focus order (atm this is inconsistent in master), so the focus goes straight from block content to content. The meta shortcut (and probably the current editor shortcut in wp-admin) should show the toolbar and move the focus to the first item, and then you can use tabbing as usual to go over the buttons. Esc to move focus back.

**Comment by jasmussen** [comment]
> So, to clarify, the goal of this PR is to ignore the toolbars entirely in the focus order (atm this is inconsistent in master), so the focus goes straight from block content to content. The meta shortcut (and probably the current editor shortcut in wp-admin) should show the toolbar and move the focus to the first item, and then you can use tabbing as usual to go over the buttons. Esc to move focus back.

This sounds right according to my reading of #297.

File: editor/modes/visual-editor/block.js [hunk 1]
```diff
@@ -104,6 +110,20 @@ class VisualEditorBlock extends wp.element.Component {
...
+
+		if ( metaKeyPressed( event ) ) {
+			this.metaCount++;
+		} else {
+			this.metaCount = 0;
+		}
+	}
+
+	handleKeyUp() {
+		if ( this.metaCount === 1 ) {
```
Comment (aduth) [inline editor/modes/visual-editor/block.js]: Maybe a bit odd to expect, but if I hold down a non-meta character (say, `c`), then press a meta character, when releasing, `metaCount` is `=== 1`.

Comment (iseulde) [inline editor/modes/visual-editor/block.js]: Thanks for testing! 👍

**Comment by iseulde** [comment]
#2960 took this over.

**Pull request closed without merging**